react/src/Components/Utils/QueryOutput.js

```diff
@@ -30,6 +30,7 @@ const QueryOutput = ({output, onArgs, onChangeTab, tabIndex}) => {
                 <Tab label="Tree view" />
                 <Tab label="JSON view" />
                 {onArgs && <Tab label="Arguments" />}
+                {showLogs && <Tab label="Logging" />}
             </Tabs>
             <Collapse in={tabIndex === 0}>
```

Comment:
I think it is a good idea to define an enumerator for the tab indexes. Something like:
```javascript
const Tab = {
   TREE: 0,
   JSON: 1,
   ARGUMENTS: 2,
   LOG: 3
};
Object.freeze(Color);
```

Reply:
see d5c611e